detectable/src/main/java/com/synopsys/integration/detectable/detectables/pnpm/lockfile/model/PnpmProjectPackagev6.java

```diff
@@ -0,0 +1,16 @@
+package com.synopsys.integration.detectable.detectables.pnpm.lockfile.model;
+
+import java.util.Map;
+
+import org.jetbrains.annotations.Nullable;
+
+public class PnpmProjectPackagev6 {
+    @Nullable
+    public Map<String, PnpmDependencyInfo> dependencies;
+
+    @Nullable
+    public Map<String, PnpmDependencyInfo> devDependencies;
+
+    @Nullable
+    public Map<String, PnpmDependencyInfo> optionalDependencies;
```

Comment:
These new PnpmDependencyInfo classes capture that fact that additional data now exists below the dependency level in v6. Specifically we capture the specifier (how the dependency was asked for ie: ^3.6.0) and the version (what was actually retrieved, ie: 3.6.1)